server/planning/planning_export_templates.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+#  Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import superdesk
+from superdesk.utils import ListCursor
+
+
+class PlanningExportTemplatesResource(superdesk.Resource):
+    endpoint_name = 'planning_export_templates'
+    schema = {
+        'name': {
+            'type': 'string',
+            'iunique': True,
+            'required': True,
+            'nullable': False,
+            'empty': False
+        },
+        'type': {
+            'type': 'string',
+            'allowed': ['event', 'planning']
+        },
+        'template': {'type': 'dict'},
+        'label': {'type': 'string'},
+    }
+    resource_methods = ['GET']
+
+
+default_export_templates = [{
+    'name': 'default_planning',
+    'label': 'Default Planning Template',
+    'type': 'planning',
+    'template': {
+        'body_html': '''
+{% for item in items %}
+<p><b>{{ item.name or item.headline or item.slugline }}</b></p>
+<p>{{ item.description_text }}</p>
+<p></p>
+{% if item.get('event', {}).get('location') %}
+<p>Location: {{ item.event.location[0].name }}.</p>
+{% endif %}
+{% if item.get('ednote', '') != '' %}
+<p>Editorial note: {{ item.ednote }}</p>
+{% endif %}
+{% if item.coverages %}
+<p>Planned coverage: {{ item.coverages | join(', ') }}
+{% endif %}
+<p>---</p>
+{% endfor %}
+'''
+    }
+}, {
+    'name': 'default_event',
+    'label': 'Default Event Template',
+    'type': 'event',
+    'template': {
+        'body_html': '''
```

Comment:
imo would be useful to be able to use actual template file for something more complex,
maybe we could have either inline `body_html` field or `body_html_template` which would be template file name and would be rendered via `flask.render_template`

Reply:
@petrjasek could you please take a look now. I have kept the default template a string to reduce new files being added, if you prefer that to be file too, I can change it.

Reply:
I would rather not have json in template, but something like:
```
{
    "name": "inline_event",
    "data": {
        "slugline": "foo",
        "body_html": "{% ... %}",
    },
},
{
    "name": "template_event",
    "data": {
        "slugline": "bar",
        "body_html_template": "export-event.template.html",
    }
}
```